src/AspNetCoreModuleV2/RequestHandler/outofprocess/outprocessapplication.cpp

```diff
@@ -101,7 +99,7 @@ OUT_OF_PROCESS_APPLICATION::CreateHandler(
     //add websocket check here
     SetWebsocketStatus(pHttpContext);
 
-    pHandler = new FORWARDING_HANDLER(pHttpContext, m_fOutOfProcessWebSocketSupported, this);
+    pHandler = new FORWARDING_HANDLER(pHttpContext, m_fWebSocketSupported, this);
```

Comment:
you can tell whether websocket is supported based on pHttpContext, you don't need to pass in m_fWebSocketSupported

Reply:
That is inefficient. We would check if websockets are enabled every request with a server variable.